dandiapi/settings.py

```diff
@@ -49,6 +49,8 @@ def before_binding(configuration: Type[ComposedConfiguration]):
 
     DANDI_DANDISETS_BUCKET_NAME = values.Value(environ_required=True)
     DANDI_DANDISETS_BUCKET_PREFIX = values.Value(default='', environ=True)
+    DANDI_ZARR_PREFIX_NAME = values.Value(defult='zarr', environ=True)
```

Comment:
default

shame on me

Reply:
nothing to be ashamed of - we are all just humans ;) for the clarity (since my eyes didn't grasp immediately the point):

```suggestion
    DANDI_ZARR_PREFIX_NAME = values.Value(default='zarr', environ=True)
```